CMakeLists.txt

```
@@ -1622,18 +1622,20 @@ if(CONFIG_BUILD_OUTPUT_BIN AND CONFIG_BUILD_OUTPUT_UF2)
 endif()
 
 if(CONFIG_BUILD_OUTPUT_META)
+  set(KERNEL_META_PATH  ${PROJECT_BINARY_DIR}/${KERNEL_NAME}.meta CACHE INTERNAL "")
```

Please keep using `KERNEL_META_NAME`, so that the consistent definition of that value is ensured from here:
https://github.com/zephyrproject-rtos/zephyr/blob/b6d9ed095d7bc395be602072db2679d201735113/cmake/modules/kernel.cmake#L168

```suggestion
  set(KERNEL_META_PATH ${PROJECT_BINARY_DIR}/${KERNEL_META_NAME} CACHE INTERNAL "")
```